packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -1,7 +1,112 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700');
+@import './variables';
+@import './text_utility';
+
+:root {
+  // stylelint-disable-next-line custom-property-pattern
+  --nav-container-width: 320px;
```

Comment:
Also, I think we are moving towards just defining defaults rather than creating the variables standalone in cases where they don't need to be shared. so just do this:
```
width: var(--terraConsumer-nav-container-width, 320px);
```